nomos-da/kzgrs-backend/src/verifier.rs

```diff
@@ -64,4 +66,39 @@ mod test {
             assert!(verifier.verify(&light_share, &commitments, domain_size));
         }
     }
+
+    // TODO: Remove this when we have the proper benches in the proofs
+    #[cfg(target_arch = "x86_64")]
+    #[expect(
+        clippy::undocumented_unsafe_blocks,
+        reason = "This test is just to measure cpu and should be run manually"
+    )]
+    #[ignore = "This test is just for calculation the cycles for the above set of proofs. This will be moved to the pertinent proof in the future."]
+    #[test]
+    fn test_verify_cycles() {
+        let iters = 500u64;
+
+        let encoder = DaEncoder::new(DaEncoderParams::default_with(2));
+        let data = rand_data(32);
+        let domain_size = 2usize;
+        let verifier = DaVerifier::new(GLOBAL_PARAMETERS.clone());
```

Comment:
Is the `clone` needed?

Reply:
Yes, otherwise it tries to move, which it can't.